docs/source/platform/schema-validation.mdx

```diff
@@ -3,27 +3,52 @@ title: Validating schema changes
+[//]: # (Assignee: Caydie)
+[//]: # (Reviewer: Jake)
+
+There are many types of schema changes that can be potentially breaking to clients, like removing a field, if made without special consideration. For safety, some organizations take the approach of _never_ making these types of changes, but this leads to an ever-growing schema and reduced API flexibility over time. In the case of federated services where multiple organizations or teams are working on different parts of the same graph, these kinds of breaking changes are especially critical to monitor. In reality, making these types of changes to a schema can be very safe as long as you have tools in place to ensure that no queries are broken in the process.
 
 Apollo provides a tool to protect for exactly this scenario called **schema validation**.
 
+// TODO modify this top-level description for how schema validation for federation is done at the **service** level
+
 > **Note:** Schema validation is an Apollo Platform feature available on the [Team and Enterprise plans](https://www.apollographql.com/plans/) of [Apollo Engine](https://engine.apollographql.com).
 
+
 ## How it works
 
 Schema validation is run through the Apollo CLI by executing the `apollo service:check` command. Apollo will generate a diff between your local schema and your most recently registered schema, then validate that the changes are safe by checking if any queries actively running against your graph will be affected.
+If your graph is composed using federation, you can also use the `apollo service:check` command to validate individual service changes by providing a service name. However, unlike running service check on a non-federated service, Engine first needs to run composition on the proposed partial schema with the current list of implementing services and their partial schemas. Composition must pass before validation can continue.
 
 Here's how it works:
 
+// [TODO: Hopefully make this a toggle, where you can see the federation and the non-federation version of how it works].
+
 1. You run `apollo service:check` locally or in CI. The proposed schema is sent to Engine's schema registry.
 1. Engine creates a diff between the local schema and the most recently published schema in the registry.
 1. Engine fetches a list of all operations sent to your graph in the last day (time window is [configurable](#adjusting-validation-parameters)).
 1. Engine walks through the schema diff change-by-change and compares against the operation list to see if the changes will affect the behavior of any operations.
 1. Engine returns the schema diff and indicates any breaking changes found.
 1. The CLI prints the output of this check with a link to view more details in the Engine UI.
 
+Here's how it works for federation:
+
+1. You _also_ run `apollo service:check` but instead you provide a service name like so: `apollo service:check --federated --serviceName=example --endpoint=http://localhost:4000/graphql`
```

Comment:
You don't need to have `--federated` anymore. I also think that we should just have an additional **extra** step in the above flow rather than repeating all the same flow here.

Reply:
oh man that's a v good idea :)